sourmash-protein/conf-HAB.yml

```diff
@@ -0,0 +1,6 @@
+# one or more files that will be combined into a single sample database
+metagenome_files:
+- ../data/marine/SLE_2018_wet.fq.gz
+
+# sourmash-specific config params
+ksize: 31
```

Comment:
note that for protein, you want `ksize: 10` here. That's the default and it's what your code is currently _using_ because `ksize` isn't used in the Snakefile; see other comments, too :)

Reply:
added ksize: 10 and scaled: 200